Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG PYTHON=3.13
+FROM python:$PYTHON-slim-bookworm
+
+ARG LMDBM=0.0.6
+RUN apt update && apt -y install build-essential && \
+	pip install lmdbm==$LMDBM pytablewriter genutility rich && \
+	apt purge --auto-remove --yes build-essential && apt clean && \
+	rm --recursive --force /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+ENV PYTHONUNBUFFERED 1
```

Comment:
1 warning found (use docker --debug to expand):
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 10)

Reply:
Changed it